applications/rapid-analysis/values-summit.yaml

```diff
@@ -10,7 +10,8 @@ env:
   LSST_RESOURCES_S3_PROFILE_butler: https://s3-butler.cp.lsst.org
   AWS_REQUEST_CHECKSUM_CALCULATION: WHEN_REQUIRED
   BUTLER_ROLLOUT_PAUSE: 1.0
-  RA_PULL_DIRECTORIES: "donut_viz ts_wep summit_utils summit_extras rubintv_production"
+  RA_PULL_DIRECTORIES: "donut_viz drp_pipe ts_wep summit_utils summit_extras rubintv_production"
+  TARTS_DATA_DIR: /project/rubintv/TARTS
```

Comment:
Is it going to break USDF to not have this set?

Reply:
Oh yes, good call. Will set that all round.